packages/design-system/src/components/Modal/Modal.tsx

```diff
@@ -41,6 +42,7 @@ const UnstyledModal: React.FC<ModalProps> = ({
   onAfterOpen,
   onAfterClose,
   contentRef,
+  disableBackgroundScroll = true,
```

Comment:
So just to confirm, this is essentially the modal behavior now. Its important that the behavior will not change for any modals currently used in our products.